get-all-users.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+# Retrieves all users in the system and outputs a CSV file.
+
+# Requires cf CLI version 7+ and jq to work.
+
+# NOTE:  Only works if there are 5,000 or less users in the system at this time.
+
+set -e -x
+
+# Check to see if the person has at least version 7 of the cf CLI; if not, let
+# them know to upgrade and exit.
+min_cf_version=7
+cf_version=$(cf version | awk '{print $3}' | cut -c 1)
+
+if [ "$cf_version" -lt "$min_cf_version" ]; then
+  echo
+  echo "You must update your cf CLI to at least version 7."
+  echo "Please run: brew update && brew upgrade cf-cli@7"
+  echo
+
+  exit 1
+fi
+
+# Retrieve all users as JSON.
+cf curl '/v3/users?per_page=5000' > all-users.json
```

Comment:
Should the script `rm` this file when done since it's only an intermediate artifact?

Reply:
It could and I thought about that; I kept it for the moment in case having the raw JSON around is still helpful for whatever reason, but that's also just a `cf curl` call away.